scripts/setup_check.sh

```diff
@@ -38,3 +32,13 @@ if ! git rev-parse origin/master >& /dev/null; then
   git remote set-branches --add origin master
   git fetch origin
 fi
+
+# Setup mint which installs a versioned swiftformat.
+set -x
+brew upgrade mint || brew install mint || true
```

Comment:
Why can't this just be `brew install mint`? If it is installed, does it need to be upgraded? If so, could you add a comment here about what's going on? (Note the difference with `clang-format`, below.)

Reply:
I made the two brew installs consistent with the old way we were doing it.